### PR TITLE
At-University-Api-009: domain for category update

### PR DESCRIPTION
**Pull request by ErnestoZamora**
domain for category update

### CODE REVIEW

File: src/main/java/com/agilethought/internship/university/service/impl/UniversityServiceImpl.java [hunk 1]
```diff
@@ -43,9 +45,31 @@ private Course requestToCourse(CreateCourseRequest request){
...
 
-        return repository.findAll();
+        List<Course> unchanged = repository.findAll();
+        List<CourseResponse> newList = new ArrayList<>();
+
+        for (Course c:unchanged) {
+            CourseResponse changed = new CourseResponse();
+
+            changed.set_id(c.get_id());
+            int indicator = c.getCategory();
```
Comment (JoseCortes-Agile) [inline src/main/java/com/agilethought/internship/university/service/impl/UniversityServiceImpl.java]: It is not necessary to create a variable, you can evaluate the category field directly in the IF

File: src/main/java/com/agilethought/internship/university/service/impl/UniversityServiceImpl.java [hunk 2]
```diff
@@ -43,9 +45,31 @@ private Course requestToCourse(CreateCourseRequest request){
...
+            if(indicator == 1)
+                changed.setCategory("JAVA");
+            else if(indicator == 2)
+                changed.setCategory("PEGA");
+            else
```
Comment (JoseCortes-Agile) [inline src/main/java/com/agilethought/internship/university/service/impl/UniversityServiceImpl.java]: Use if = 3 to evaluate JS category and add "UNKNOWN" for others categories at the end

**Review by JoseCortes-Agile — CHANGES_REQUESTED** [review]

File: src/main/java/com/agilethought/internship/university/service/impl/UniversityServiceImpl.java (hunk 1 above)
Comment (ErnestoZamora) [reply src/main/java/com/agilethought/internship/university/service/impl/UniversityServiceImpl.java]: done

File: src/main/java/com/agilethought/internship/university/service/impl/UniversityServiceImpl.java (hunk 2 above)
Comment (ErnestoZamora) [reply src/main/java/com/agilethought/internship/university/service/impl/UniversityServiceImpl.java]: done

**Review by JoseCortes-Agile — APPROVED** [review]

**Review by Alberto-R25 — APPROVED** [review]

**Review by JoseLazcano-JLo — APPROVED** [review]

**Pull request merged**